perception/sub8_perception/sub8_vision_tools/rviz.py

```diff
@@ -14,8 +14,8 @@ class RvizVisualizer(object):
     '''Cute tool for drawing both depth and height-from-bottom in RVIZ
     '''
 
-    def __init__(self):
-        self.rviz_pub = rospy.Publisher("visualization/projection", visualization_msgs.Marker, queue_size=3)
+    def __init__(self, topic="visualization/projection"):
```

Comment:
I disagree about publishing all markers to the same topic, unless everyone publishes to both a specific and a general topic. It's often very useful to reduce clutter as much as possible when visualizing.

This is ok though.